spec/Section 5 -- Validation.md

```diff
@@ -1431,6 +1452,103 @@ arguments, an input object may have required fields. An input field is required
 if it has a non-null type and does not have a default value. Otherwise, the
 input object field is optional.
 
+### OneOf Input Objects Have Exactly One Field
+
+**Formal Specification**
+
+- For each {operation} in {document}:
+  - Let {oneofInputObjects} be all OneOf Input Objects transitively included in
+    the {operation}.
+  - For each {oneofInputObject} in {oneofInputObjects}:
+    - Let {fields} be the fields provided by {oneofInputObject}.
```

Comment:
Nit: since `fields` has more of an "output" connotation, should we use `inputFields` instead? 

```suggestion
    - Let {inputFields} be the input fields provided by {oneofInputObject}.
```

Reply:
Not in this PR; the spec refers to input object fields as `fields` (rather than `inputFields`) in many places; for example: https://spec.graphql.org/draft/#sel-IALXJDDBBDBAACDAn-P - in fact, none of the algorithms in the spec refer to them as `inputFields` so this would be the first instance. For now we should maintain consistency.

Reply:
Makes sense 👍 Thanks for the context!